### PR TITLE
US17450 - Shared Header, Footer, and Snail Trail

### PR DESCRIPTION
**Pull request by J-Byron**
## Problem
        * Current Shared-header/shared-footer approach has been depreciated

## Solution
        * Update the Shared-header/Shared-footer with crds-components

## Testing
        * Netlify preview:

### CODE REVIEW

File: _assets/stylesheets/_cards.scss [hunk 1]
```diff
@@ -118,12 +118,6 @@
   }
 }
 
-.card-block {
-  .component-header {
-    line-height: 0.95;
-  }
```
Comment (seancdavis) [inline _assets/stylesheets/_cards.scss]: Why deleting this? This looks like it's part of the card.

File: _assets/stylesheets/_jumbotron.scss [hunk 2]
```diff
@@ -78,47 +78,6 @@
   }
 }
 
-.header-content  {
```
Comment (seancdavis) [inline _assets/stylesheets/_jumbotron.scss]: This looks like it's jumbotron content, not part of the shared header.

File: _includes/_footer.html [hunk 3]
```diff
@@ -1,25 +1,3 @@
...
-  <a href="https://www.crossroads.net/app/">Get the Crossroads App</a>
-  </p>
-</div>
-<div class="col-sm-4 social-icons"><a href="https://twitter.com/crdschurch/" target="_blank" class="text-center"><img src="//d1tmclqz61gqwd.cloudfront.net/images/twitter.svg" alt="twitter account" title="twitter account"></a> <a href="https://www.facebook.com/crdschurch/?_rdr=p" target="_blank" class="text-center"> <img src="//d1tmclqz61gqwd.cloudfront.net/images/facebook.svg" alt="facebook account" title="facebook account"></a> <a href="https://www.youtube.com/user/crdschurch" target="_blank" class="text-center"> <img src="//d1tmclqz61gqwd.cloudfront.net/images/youtube.svg" alt="youtube account" title="youtube account"></a> <a href="https://www.instagram.com/crdschurch/" target="_blank" class="text-center"> <img src="//d1tmclqz61gqwd.cloudfront.net/images/instagram.svg" alt="instagram account" title="instagram account"></a></div>
-</div>
+<div class="shared-footer-skeleton" style="min-height: 727px; background-color: #e7e7e7; position: relative;">
```
Comment (tcmacdonald) [inline _includes/_footer.html]: Please update crds-styles to the latest version and remove these inline styles.

File: _includes/header.html [hunk 4]
```diff
@@ -1,8 +1,9 @@
-<div class="mobile-header visible-xs-block">
-  {% include _mobile-header.html %}
+<div class="shared-header-skeleton" style="min-height: 49px; background-color: #0095d9; position: relative; z-index: 9999;">
```
Comment (tcmacdonald) [inline _includes/header.html]: More inline styles we need to remove.

**Pull request merged**